### ISSUE TITLE
Open TV ISDB-T Receiver

### ISSUE THREAD

**Issue by laercionit** [xibosignage/xibo#49]
Open TV ISDB-T Receiver

There are many softwares on the market that have content management and inclusion of broadcast TV on your schedule. If we use a USB Digital TV receiver in android, we could view the contents with broadcast TV.

This innovation already present in some other products and market software.

**Comment by dasgarner** [comment]
I think this applies to the open source players and to Android and could be described more generically (open capture device)

**Comment by nagyrobi** [comment]
+1 for this, we had a request for this which we had to abandon

**Comment by csantiba** [comment]
+1 for this, we have an user request.

The ISDB-T Receiver output content should be presented as a region of a layout
for which we should consider incorporating a module that is responsible for this function.

**Comment by andrew867** [comment]
+1 From me as well.

Also it would be great if the video decoder/player module could support HTTP and RTP/UDP streams as well. I know VLC/libvlc can open DirectShow on Windows, V4L on Linux and any network stream on either platform.

The support for RTP/UDP would be great for large networks that want to play multicast streams (think synchronized screens throughout a store/building).

**Comment by nagyrobi** [comment]
Yes indeed. This should be possible in Android too, and supporting servers like Live555 (see http://www.live555.com/) would be great.

**Comment by dasgarner** [comment]
If you have a RTSP stream already you can put it as the source for the Local Video module.

I don't think the same will work with the local capture card.

**Comment by nagyrobi** [comment]
Really? How? Using Ubuntu Linux Server 14.04.

**Comment by dasgarner** [comment]
Make sure the local video module is enabled - add a local video module to your layout and in the video path put the stream URL.

The stream will need to be accessible from the player and in a codec that the player supports.
